src/Mobs/Path.cpp

```diff
@@ -51,6 +51,14 @@ cPath::cPath(
 	m_Source.x = FloorC(a_StartingPoint.x - HalfWidthInt);
 	m_Source.y = FloorC(a_StartingPoint.y);
 	m_Source.z = FloorC(a_StartingPoint.z - HalfWidthInt);
+	
+	if ((m_Source.y <= 0) || (m_Destination.y <= 0)
+		|| (m_Source.y > 256) || (m_Destination.y > 256))
```

Comment:
>That's a bad assumption - it would mean that mobs would try to walk on top of the void in worlds where there's no bedrock (End, SkyBlock etc.)

That's a very good point.